### ISSUE TITLE
Support more than 1024 pods via multiple Linux bridges

### ISSUE THREAD

**Issue by jokerwyt** [flannel-io/flannel#2095]
For now, Linux hardcodes a limit (1024) for the maximum ports of an eth bridge. The kernel recompilation is needed to make it larger. Therefore, flannel cannot support pods of more than 1024. I think we can use multiple bridges to overcome this problem. We connect them together and assign pods' veth to one of them. Does that solution make sense?

**Comment by tanvp112** [comment]
Hi there, if not mistaken this limitation is on a single node right?

**Comment by jokerwyt** [comment]
> Considering that by default for IPv4 the masklen is /24 for the node CIDR then the number of pods that will be around 250 more or less. Your case requires a specific configuration I don't know if using multiple bridges could be the best solution.

Yeah but putting more than 256 pods in a node is not a rare need according to my experience
